hooks.md

```diff
@@ -1,87 +1,54 @@
 # OCI Hooks Configuration
 
-[The OCI Runtime Specification defines POSIX-platform Hooks:](
-https://github.com/opencontainers/runtime-spec/blob/master/config.md#posix-platform-hooks)
+For POSIX platforms, the [OCI runtime configuration][runtime-spec] supports [hooks][spec-hooks] for configuring custom actions related to the life cycle of the container.
+The way you enable the hooks above is by editing the OCI runtime configuration before running the OCI runtime (e.g. [`runc`][runc]).
+CRI-O and `Kpod create` create the OCI do for you, and this documentation allows developers to configure CRI-O to set their intended hooks.
```

Comment:
`..create the OCI do for you..` seems incorrect.

Reply:
> `..create the OCI do for you..` seems incorrect.

Thanks.  Fixed in 60a419868 → cd173dd5e.